Sources/Purchasing/Offerings.swift

```diff
@@ -55,7 +55,10 @@ import Foundation
         return all[currentOfferingID]?.copyWith(targeting: self.targeting)
     }
 
-    internal let response: OfferingsResponse
+    internal var response: OfferingsResponse {
+        return self.contents.response
+    }
+    internal let contents: Offerings.Contents
```

Comment:
I missed this in Android, so just to check here, should we also ignore the new fields in equality checks? Mostly to avoid any hidden changes in the developers side.

Reply:
Yep. I think we should totally ignore these new fields. In fact, in iOS, `Offerings` is not `Equatable`, so with `==` you only get instance equality...